### DISCUSSION TITLE
Could pyroscope support tracking python memory utilization?

### DISCUSSION THREAD

**Discussion by sysudengle** [grafana/pyroscope#2760] [Q&A]
I have installed my pyroscope for latest version within my k8s cluster. And then I have a python flask app running and reporting tracking data to pyroscope. The result was that there were only cpu stuff for my python app. 
So my question is, could pyroscope support tracking py memory utilization? I have tried to find the demo in the main page but still could not find any memory stuff. Could anyone help answering? Thanks!

**Comment by korniltsev** [comment]
Not at the moment

**Comment by Rperry2174** [comment]
There is an issue here https://github.com/grafana/pyroscope/issues/316 basically how all of our integrations work is that we typically wrap a popular open source profiler. 

* [x] Go (via `pprof`)
* [x] Python (via `py-spy`)
* [x] Ruby (via `rbspy`)
* [x] Linux eBPF
* [x] Java (via `async-profiler`)
* [x] Rust (via `pprof-rs`)
* [x] .NET
* [x] PHP (via `phpspy`)
* [x] Node

so basically if you know of a good open source memory profiler for Python we could make a few modifications to be able to ingest it into Pyroscope in the right format.
